### PR TITLE
wip - fix up broken nightly tests

### LINKED ISSUE #207: Failing nightly build - Rack is unhappy
**Versions**

- Ruby: all the versions in the test matrix
- Beeline: [main](https://github.com/honeycombio/beeline-ruby/tree/c82a29dd0c8bf75ddafe4bc05d0ee1161fe7b0b9) as of c82a29dd0c8bf75ddafe4bc05d0ee1161fe7b0b9

**Steps to reproduce**

1. See the Rack failures in [a sad nightly build](https://app.circleci.com/pipelines/github/honeycombio/beeline-ruby/1511/workflows/aeea2e4f-6a25-4122-a374-23bf37a37449)

**Additional context**

### PR DESCRIPTION
**Pull request by JamieDanielson**
## Which problem is this PR solving?

- closes #207 

## Short description of the changes

-

### CODE REVIEW

File: spec/honeycomb/integrations/rack_spec.rb [hunk 1]
```diff
@@ -1,15 +1,26 @@
 # frozen_string_literal: true
 
 if defined?(Honeycomb::Rack)
+  require "rack"
   require "rack/test"
-  require "rack/lobster"
```
Comment (JamieDanielson) [inline spec/honeycomb/integrations/rack_spec.rb:5]: rack lobster moved to a new gem

File: gemfiles/rack.gemfile [hunk 2]
```diff
@@ -3,6 +3,7 @@
 source "https://rubygems.org"
 
 gem "rack"
+gem "rack-session"
```
Comment (JamieDanielson) [inline gemfiles/rack.gemfile:6]: rack session moved to a new gem

File: spec/honeycomb/integrations/rack_spec.rb [hunk 3]
```diff
@@ -1,15 +1,26 @@
...
   require "warden"
+  require "rack/session"
 
   RSpec.describe Honeycomb::Rack do
     include Rack::Test::Methods
     let(:libhoney_client) { Libhoney::TestClient.new }
     let(:event_data) { libhoney_client.events.map(&:data) }
-    let(:lobster) { Rack::Lobster.new }
+
+    # rubocop:disable Lint/UnusedMethodArgument
```
Comment (JamieDanielson) [inline spec/honeycomb/integrations/rack_spec.rb:14]: argument (env) is required even though rubocop argues that it's not used

**Comment by JamieDanielson** [comment]
Closing as Em's got this better handled in #209

**Comment by emilyashley** [comment]
We just pulled in your changes to remove Lobster, too! d8b881863bb9dad3d7222707ff4f8fbc35385f09

**Pull request closed without merging**